lxd/container.go

```diff
@@ -412,6 +425,7 @@ type container interface {
 	LogPath() string
 
 	// FIXME: Those should be internal functions
+	StoragePool() string
```

Comment:
The name of the storage pool seems appropriate public property for a container, so shouldn't be listed in the to-be-removed list.